core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -2658,6 +2692,7 @@ String exec() {
         for (Function<RelNode, RelNode> transform : transforms) {
           rel = transform.apply(rel);
         }
+        System.out.println(RelOptUtil.toString(rel));
```

Comment:
Minor, but looks like this was for debugging and should be removed.

Reply:
Sorry, I forgot to remove it